ulogger/stackdriver.py

```diff
@@ -20,15 +20,16 @@
 import logging
 
 import requests
-from google.cloud import logging as gcl_logging
-from google.cloud.logging import handlers as gcl_handlers
-from google.cloud.logging import resource as gcl_resource
+from google.cloud import logging_v2 as gcl_logging
+from google.cloud.logging_v2 import handlers as gcl_handlers
+from google.cloud.logging_v2 import resource as gcl_resource
```

Comment:
This was the only change actually needed to get tox to pass but I figured I should make it consistent